data/brands/shop/supermarket.json

```
@@ -3834,6 +3834,18 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Longo's",
+      "id": "longos-76454b",
+      "locationSet": {"include": ["ca"]},
```

It [looks like](https://www.longos.com/store-finder)  they only have locations in Ontario so I would refine the locationSet to only include that province
```suggestion
      "locationSet": {"include": ["ca-on.geojson"]},
```